docs/csharp/language-reference/compiler-options/langversion-compiler-option.md

```diff
@@ -90,10 +89,11 @@ Causes the compiler to accept only syntax that is included in the chosen C# lang
 [↩](#TCS4)<a name="FCS4">CS4</a>: Microsoft Visual Studio/Build Tools 2010 or bundled .Net Framework 4.0 compiler    
 [↩](#TCS5)<a name="FCS5">CS5</a>: Microsoft Visual Studio/Build Tools 2012 or bundled .Net Framework 4.5 compiler    
 [↩](#TCS6)<a name="FCS6">CS6</a>: Microsoft Visual Studio/Build Tools 2015    
-[↩](#TCS7)<a name="FCS7">CS7</a>, <a name="FLatest">Latest</a>: Microsoft Visual Studio/Build Tools 2017   
+[↩](#TCS7)<a name="FCS7">CS7</a>: Microsoft Visual Studio/Build Tools 2017   
+[↩](#TCS71)<a name="FCS71">CS71</a>: Microsoft Visual Studio/Build Tools 2017    
+[↩](#TCS72)<a name="FCS72">CS72</a>: Microsoft Visual Studio/Build Tools 2017    
+[↩](#TCS73)<a name="FCS73">CS73</a>, <a name="FLatest">Latest</a>: Microsoft Visual Studio/Build Tools 2017    
```

Comment:
I believe the 7.x versions require specific minor versions of VS 2017 (like 15.7 for C# 7.3). Is that worth specifying here? If not, you can probably combine all the VS 2017 lines into one.

Reply:
Another question: what should be told to the folks using Visual Studio Code with .NET Core (so, no Visual Studio)?

Reply:
@pkulikov I think that for .Net Core, you wouldn't use the `-langversion` parameter to csc, you would use the `<LangVersion>` property in csproj. So, this page isn't really relevant to .Net Core.

Reply:
Isn't the documentation for the csproj property (by the way, is it up-to-date itself?) the same as for this compiler option? The related Visual Studio doc page references this compiler option page.

Reply:
I'm talking about this page:
docs.microsoft.com/en-us/visualstudio/ide/reference/advanced-build-settings-dialog-box-csharp

Reply:
@pkulikov I believe `<LangVersion>` in csproj is currently not documented: https://github.com/dotnet/docs/issues/2821.

Reply:
I think we should say which version of the .NET core tools is required for the C# versions 6.0 and greater (if my timeline is right).

With #2821 I'd like to put that information in one easy-to-find place. Until then, it should be here.

Reply:
@BillWagner I've added required Visual Studio 2017 versions for C# 7.1...7.3 versions. As for .NET Core, it's related to #4154. Is there a mapping .NET Core SDK version -> Roslyn version? Then, from the Roslyn version, we [can get the language version](https://github.com/dotnet/roslyn/wiki/NuGet-packages#versioning)

Reply:
That PR might be useful: dotnet/core#1454